TheEmpireStrikesBack/TheEmpireStrikesBack/Controller/ViewController.swift

```diff
@@ -39,9 +42,19 @@ class ViewController: UIViewController {
                 print(error)
                 return
             }
-            if let responseData = data {
-                print("Data Retrieved.  Amount:")
-                print(responseData)
+            guard let data = data else {
+                return
+            }
+            print("Data Retrieved. Amount:")
+            print(data)
```

Comment:
Are these print statements still useful to you? If so, keep them, but if not, delete them.